test/systems/plonk/placeholder/placeholder.cpp

```diff
@@ -615,13 +622,17 @@ ACTOR_THREAD_TEST_CASE(permutation_argument_test) {
     for (int i = 0; i < argument_size; i++) {
         BOOST_CHECK(prover_res.F_dfs[i].evaluate(y) == verifier_res[i]);
         for (std::size_t j = 0; j < desc.rows_amount; j++) {
-            BOOST_CHECK(
-                prover_res.F_dfs[i].evaluate(preprocessed_public_data.common_data.basic_domain->get_domain_element(j)) == field_type::value_type::zero());
+            if(prover_res.F_dfs[i].evaluate(preprocessed_public_data.common_data.basic_domain->get_domain_element(j)) != 0){
```

Comment:
It's something interesting. 
We always check inside prover values with `field_type::value_type::zero()`
Such case means - our equation not always correct?

Reply:
It seems that some debug checks creeped in.
fixed in c1c33e86